src/core/store.js

```diff
@@ -30,11 +30,13 @@ const PROMISE_PREFIXES = {
 export function middleware({
   _config = config, _createLogger = createLogger,
   _window = typeof window !== 'undefined' ? window : null,
+  sagaMiddleware = () => { return (next) => (action) => next(action); },
```

Comment:
This seems awkward to me. Why not just have it accept arbitrary additions like this?

````js
export function middleware({
  _config = config, _createLogger = createLogger,
  _window = typeof window !== 'undefined' ? window : null,
  additions = [],
} = {}) {
  if (_config.get('isDevelopment')) {
    return compose(
      applyMiddleware(
        _createLogger(),
        ...additions,
        loadingBarMiddleware(PROMISE_PREFIXES),
      ),
      _window && _window.devToolsExtension ?
        _window.devToolsExtension() : (createStore) => createStore
    );
  }

  return compose(
    applyMiddleware(
      ...additions,
      loadingBarMiddleware(PROMISE_PREFIXES),
    ),
    (createStore) => createStore,
  );
}
````